src/entities/tokens.ts

```diff
@@ -40,6 +40,7 @@ export interface ParsedAddressAsset extends Asset {
     value?: number;
   };
   type?: string;
+  id: string;
```

Comment:
A few places in the web3 handler imply that this is defined, and it is also set to a string in `parseAccountUniqueTokens`.

Reply:
Can we make a separate Linear ticket to keep track of potentially splitting up `ParsedAddressAsset` from UniqueToken data (which is used for NFTs and has the `asset_contract` you added here)

Reply:
Definitely, I added RNBW-1879 for this.